### ISSUE TITLE
how to use other models?

### ISSUE THREAD

**Issue by jerryhero** [prakashpandey9/Text-Classification-Pytorch#1]
I find that "main.py" only support "LSTM". I want to know how to use other models to make text classification,such as RNN,CNN,RCNN,attention,self-attention.I find other models need parameter "weight" which is not found in "main.py".How to set the "weight" parameter to call the other models?
Thanks a lot!

**Comment by prakashpandey9** [comment]
@jerryhero  You can just import those models from models directory as:
 `from models.selfAttention import SelfAttention`
Except for CNN model, the arguments of all models are same and "weights" are just pretrained parameters of Glove model for word embeddings which is returned when we are calling load_data.load_dataset() (line 11 of main.py). So weights = word_embeddings. 

Arguments of CNN model is as follows : in_channels=1, out_channels=128, kernel_heights=[3, 4, 5], stride=1, padding=0, keep_probab=0.8 . 
Let me know if you need further help. Thanks
